leaper.py

```diff
@@ -135,7 +135,7 @@ def rdiff_link(self, src_project, src_package, src_rev, target_project, target_p
                 }
 
     def check_source_submission(self, src_project, src_package, src_rev, target_project, target_package):
-        self.logger.info("%s/%s@%s -> %s/%s"%(src_project, src_package, src_rev, target_project, target_package))
+        super(Leaper, self).check_source_submission(src_project, src_package, src_rev, target_project, target_package)
```

Comment:
IMHO in python3 you can just leave out the argument. For python2 compatibility what can work:

```python
super(type(self), self).…
```
-> https://rhettinger.wordpress.com/2011/05/26/super-considered-super/#comment-32